src/mlpack/methods/range_search/rs_model_impl.hpp

```diff
@@ -271,7 +288,7 @@ void MonoSearchVisitor::operator()(RSType* rs) const
 }
 
 //! Save parameters for bichromatic range search.
-BiSearchVisitor::BiSearchVisitor(const arma::mat& querySet,
+inline BiSearchVisitor::BiSearchVisitor(const arma::mat& querySet,
```

Comment:
Thanks for these fixes.  Can you adjust the spacing for the other parameters to these methods to make them match where the `const arma::mat& querySet` parameter starts also please?

Reply:
The spacing is still not right here; why is it different for the `std::vector<...` lines?